blockchain/stakeversion_test.go

```diff
@@ -5,7 +5,10 @@
 package blockchain
 
 import (
+	"fmt"
 	"testing"
+	"time"
+	"unsafe"
```

Comment:
Please don't use unsafe.  We want to be able to work in the GAE environment which forbids unsafe.  Yes, I'm aware this means you can't get the size of the types used below so that will need to be removed.  If the memory usage statistics are required, there are a multitude of memory profiling flags that can be used when running the tests.  Not only is that the correct way to do memory profiling, it's much more accurate.